Modules/rat_command.py

```diff
@@ -129,6 +132,9 @@ def _register(cls, func, names: list or str) -> bool:
         if isinstance(names, str):
             names = [names]  # idiot proofing
 
+        # transform commands to lowercase
+        names = [lowered_name.lower() for lowered_name in names]
```

Comment:
Hey, this is a really tiny nitpick and also the last one I could find, but since you're lowering it only now, `lowered_name` seems unfitting.

Reply:
methinks ```py
names = [name.lower() for name in names
```
would be more appropriate

Reply:
Yeah